Messages/Message.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace Messages
+{
+    abstract public class Message
+    {
+
+        public string Subject = "Hello";
+
+        abstract public string Recipient { get; set; }
```

Comment:
Please justify the use of abstraction for a property by showing examples of its different uses in inherited classes